internal/gmailclient/real.go

```
@@ -29,6 +29,17 @@ type RealClient struct {
 	service         *gmail.Service
 }
 
+func (r *RealClient) AccountEmail() (string, error) {
+	profile, err := r.service.Users.GetProfile("me").Do()
+	if err != nil {
+		return "", fmt.Errorf("get Gmail profile: %w", err)
+	}
+	if profile.EmailAddress == "" {
+		return "", errors.New("Gmail profile returned no account identity")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)